src/components/panels/ControlPanelExtruder.vue

```diff
@@ -121,7 +121,7 @@ export default class ControlPanelExtruder extends Mixins(BaseMixin) {
     }
 
     get minExtrudeTemp() {
-        return this.$store.state.printer.configfile.config.extruder.min_extrude_temp
+        return this.$store.state.printer.configfile.config.extruder.min_extrude_temp ?? 170
```

Comment:
pls use printer.configfile.settings instead of config.

config = user config file
settings = parsed and filled with all default settings

Reply:
Thanks, fixed